functions/create_draft/interactivity_handler.ts

```diff
@@ -73,6 +74,40 @@ export const openDraftEditView: BlockActionHandler<
       });
     }
   }
+  // If the user selects to discard the draft message
+  if (action.selected_option.value == "discard_message_overflow") {
+    const id = action.block_id;
+    const thread_ts = body.message?.ts || "";
```

Comment:
Can this be called `message_ts` or `ts` instead? I don't think these messages are threaded after a bit of searching around 🔍  But please catch me if that's wrong!

Reply:
Ah yes totally!  Good call out! 👍